tests/integration/ceph_base_object_test.go

```diff
@@ -192,7 +192,11 @@ func assertObjectStoreDeletion(t *testing.T, k8sh *utils.K8sHelper, namespace, s
 	sleepTime := 3 * time.Second
 	for i = 0; i < retry; i++ {
 		storeStr, err := k8sh.GetResource("-n", namespace, "CephObjectStore", storeName, "-o", "json")
```

Comment:
We can also used kubectl's flag:

> ```
>     --ignore-not-found=false:
        Treat "resource not found" as a successful delete. Defaults to "true" when --all is specified.
> ```

Reply:
We can't use this directly because the next lines of code depends on this output,
SO we need to return back to the original call